seqr/views/apis/summary_data_api.py

```diff
@@ -162,3 +167,98 @@ def bulk_update_family_analysed_by(request):
         'warnings': warnings,
         'info': [f'Updated "analysed by" for {len(analysed_by_models)} families'],
     })
+
+
+@login_and_policies_required
+def sample_metadata_export(request, project_guid):
+    is_analyst = user_is_analyst(request.user)
+    is_all_projects = project_guid == 'all'
+    include_airtable = 'true' in request.GET.get('includeAirtable', '') and is_analyst and not is_all_projects
+    if is_all_projects:
+        projects = get_internal_projects() if is_analyst else Project.objects.filter(
+            guid__in=get_project_guids_user_can_view(request.user))
+    elif project_guid == 'gregor':
+        if not is_analyst:
+            raise PermissionDenied()
+        projects = Project.objects.filter(projectcategory__name__iexact='gregor')
+    else:
+        projects = [get_project_and_check_permissions(project_guid, request.user)]
+
+    family_rows_by_id = {}
```

Comment:
fom here down is copy-past from report_api with only one change that I'll indicate